src/settings.ts

```diff
@@ -56,6 +56,7 @@ export const DEFAULT_AVATAR_IMAGE = sparkleNavLogo;
 export const DEFAULT_PROFILE_PIC = "/default-profile-pic.png";
 export const DEFAULT_MAP_ICON_URL = defaultMapIcon;
 export const SPARKLEVERSE_LOGO_URL = sparkleverseLogo;
+export const MAP_ZOOM_MULTIPLIER = 500;
```

Comment:
What's this?

Reply:
It's a value that gets multiplied by the selected zoom level to create zoomable experience of browsing the venue map
It's kind of arbitrary since I took the `2000px` as a max zoom judging on Thomas' comment in the issue description https://github.com/sparkletown/internal-sparkle-issues/issues/718